### ISSUE TITLE
Fix instructions to run athens as docker image, unable to find the gomods/athens:latest

### ISSUE THREAD

**Issue by prateeknayak** [gomods/athens#1001]
**Describe the bug**
I was trying to run athens locally using docker and the instruction details to use the latest tag of the image. Unfortunately, It fails with `Error response from daemon: manifest for gomods/athens:latest not found`. Also checked docker hub and  I can't see the latest tag on repo. 

Looking at the travis script latest is only published when there isa tagged release. Would it be worth pushing a tagged release so that the instructions would work ? or may be updating the instructions to not use latest.

**Error Message**
<pre>
```console
docker pull gomods/athens
Using default tag: latest
Error response from daemon: manifest for gomods/athens:latest not found
```
</pre>

**To Reproduce**
Follow the instructions on https://docs.gomods.io/install/shared-team-instance/#running-athens-with-local-disk-storage

<pre>
```console
export ATHENS_STORAGE=~/athens-storage
mkdir -p $ATHENS_STORAGE
docker run -d -v $ATHENS_STORAGE:/var/lib/athens \
   -e ATHENS_DISK_STORAGE_ROOT=/var/lib/athens \
   -e ATHENS_STORAGE_TYPE=disk \
   --name athens-proxy \
   --restart always \
   -p 3000:3000 \
   gomods/athens:latest
```
</pre>

**Expected behavior**
To run athens as a docker container

**Comment by t-tomalak** [comment]
Hi @prateeknayak, it's know issue :slightly_frowning_face: We already trying to address that (https://github.com/gomods/athens/pull/989). Until this you can either use `:v0.2.0` (stable beta release) or `:canary` ( up to date with our `master` branch) tag

**Comment by prateeknayak** [comment]
@t-tomalak thanks for the prompt reply it's good to know that you are already on it. I am already using the v0.2.0, i just raised it so that it can be fixed in future for other folks.

Cheers !!

**Comment by hellojukay** [comment]
when i use v0.2.0
```txt
3.info: dial tcp 127.0.0.1:3001: connect: connection refused
```

**Comment by chriscoffee** [comment]
Hey @prateeknayak we're now tagging `latest` on master builds. We've also released v0.3.0 if you want to pull the latest image 👍 

@hellojukay did you manage to resolve your problem?

**Comment by arschles** [comment]
@hellojukay @PascalDierich now that we have `latest` published, our documentation at https://docs.gomods.io/install/shared-team-instance/#running-athens-with-local-disk-storage works now.

I am going to close this issue, but please comment here, open a new issue, or come talk to us on the Athens slack channel if you have more questions or comments!
